tests/PSRule.Rules.Azure.Tests/Azure.VMSS.Tests.ps1

```diff
@@ -233,5 +233,26 @@ Describe 'Azure.VMSS' -Tag 'VMSS' {
             $ruleResult | Should -Not -BeNullOrEmpty;
             $ruleResult.Outcome | Should -Be 'Fail';
         }
+
+        It 'Azure.VMSS.PublicKey' {
+            $dataPath = Join-Path -Path $here -ChildPath 'Resources.VMSS.json';
+            $result = Invoke-PSRule @invokeParams -InputPath $dataPath
+            $filteredResult = $result | Where-Object { $_.RuleName -eq 'Azure.VMSS.PublicKey' };
+
+            # Fail
+            $ruleResult = @($filteredResult | Where-Object { $_.Outcome -eq 'Fail' });
+            $ruleResult | Should -Not -BeNullOrEmpty;
+            $ruleResult.Length | Should -Be 2;
+            $ruleResult.TargetName | Should -BeIn 'vmss-002', 'vmss-003';
```

Comment:
Test case for `vmss-002` and `vmss-003` look the same. Maybe `vmss-003` should not have the `disablePasswordAuthentication` property.